### ISSUE TITLE
High CPU Usage when idle

### ISSUE THREAD

**Issue by hykilpikonna** [nextcloud/spreed-screensharing-chrome-extension#6]
I was using spreed for text communication, and I previously installed this screen sharing extension for testing. The CPU usage went up to about 60% after I joined and idled in a text conversation, and I was not even in a video call or screen sharing session. 

Other people with this problem: https://github.com/nextcloud/spreed/issues/906#issuecomment-477739084

OS: Windows 10.0.17763.503
Browser: Chrome 74.0.3729.157
CPU: i7-6700HQ

**Comment by nickvergessen** [comment]
@fancycode I can not explain this. The only thing the extension does is add the URL to the white list, right?

**Comment by fancycode** [comment]
No, this was the old Firefox extension. For Chrome, we insert a small JavaScript code in each tab to initialize the extension if the tab is running Nextcloud Talk:
https://github.com/nextcloud/spreed-screensharing-chrome-extension/blob/master/extension/detector.js

This howerer is only executed once for each tab when a new document is loaded, so it shouldn't cause a constant CPU load. Do you see any messages in your browser console (or the console of the extension)?
